package.json

```diff
@@ -10,8 +10,7 @@
   },
   "scripts": {
     "build": "npm run build:es2015; npm run build:cmjs",
-    "build:es2015":
-      "tsc -P ./tsconfig-build.json --outDir 'dist/es' --target es2015 --module es2015",
+    "build:es2015": "tsc -P ./tsconfig-build.json --outDir 'dist/es' --target es2015 --module es2015",
```

Comment:
`tsconfig-build.json` targets es2017 but the option here changes it to es2015? So AFAIKT the CommonJS build gets compiled to es2017 but the ES builds gets compiled to es2015?

Reply:
Thanks. This is also on master. Will fix 👍